src/tasks/minions/questingActivity.ts

```diff
@@ -40,7 +40,7 @@ export default class extends Task {
 
 		await user.addQP(qpRecieved);
 		const herbLevel = user.skillLevel(SkillsEnum.Herblore);
-		if (herbLevel === 0 && currentQP + qpRecieved > 5 && roll(2)) {
+		if ((herbLevel === 1 || herbLevel === 0) && currentQP + qpRecieved > 5 && roll(2)) {
```

Comment:
you fixed it but youre still keeping in the impossible condition, also please name your PR properly.